resources/assets/lib/beatmap-discussions/beatmap-owner-editor.tsx

```diff
@@ -117,7 +117,7 @@ export default class BeatmapOwnerEditor extends React.Component<Props> {
   private handleUsernameInput = (e: React.ChangeEvent<HTMLInputElement>) => {
     this.inputUsername = e.currentTarget.value;
 
-    this.inputUser = this.props.userByName.get(this.inputUsername);
+    this.inputUser = this.props.userByName.get(this.inputUsername.trim());
```

Comment:
that's not where the actual lookup happens... (it's in `userLookup` function, as parameter to the username check)

Reply:
this place?
```this.xhr.userLookup = $.ajax(route('users.check-username-exists'), {
      data: { username: this.inputUsername.trim() },
      method: 'POST',
    })```

Reply:
yes